neo/io/axonio.py

```diff
@@ -296,7 +296,7 @@ def read_block(self, lazy=False, cascade=True):
 
                     anaSig = AnalogSignal(signal, sampling_rate=sampling_rate,
                                           t_start=t_start,
-                                          name=name.decode('utf-8'),
+                                          name=str(name),

```

Comment:
I'm not sure what the restrictions are on the type of `AnalogSignal.name`. The tests require it to be of type `str`, which means effectively ASCII for Python 2 and unicode for Python 3. Couldn't we allow unicode in Python 2?